### DISCUSSION TITLE
TempBan Cool Down on Specific Names

### DISCUSSION THREAD

**Discussion by hamcoremusic** [Zrips/CMI#2044] [Q&A]
**Question:**
Hey! Is there any way to set a cooldown on the `/tempban` command in a way that only triggers with the same name?

I.E: I do `/tempban HAMCORE` then 10m later I do `/tempban HAMCORE` again, I'd want it to trigger a cooldown. BUT if I do a different name, it will be a seperate cooldown all together.

**Thoughts:**
I think it might be possible...

```
Cooldowns:
  - cmi tempban [playerName]
```

Do variables work in the "Cooldowns" section?

**Comment by CloudeLecaw** [comment]
no this would need to much counters for all possible commands
you just can setup for special variable (fixed one)
or for all variabled (fixed one) 
or for the command itself (fixed one)
